libraries/CurieBLE/src/internal/BLECallbacks.cpp

```diff
@@ -77,7 +77,13 @@ ssize_t profile_longwrite_process(struct bt_conn *conn,
                                      const void *buf, uint16_t len,
                                      uint16_t offset)
 {
-    BLECharacteristicImp *blecharacteritic = (BLECharacteristicImp*)attr->user_data;
+    BLEAttribute *bleattr = (BLEAttribute *)attr->user_data;
```

Comment:
I think we can use assert. Because this struct not expose to user. The CurieBLE library own this memory. How about this?